src/frontend/packages/lib_common/src/queries/deleteOne.ts

```diff
@@ -1,10 +1,12 @@
 import { useJwt } from 'lib-components';
-import { MutationFunction } from 'react-query';
 
-export const deleteOne: MutationFunction<
-  any,
-  { name: string; id: string }
-> = async ({ name, id }) => {
+export const deleteOne = async <T>({
+  name,
+  id,
+}: {
+  name: string;
+  id: string;
+}): Promise<T> => {
```

Comment:
```suggestion
}): Promise<undefined> => {
```

Reply:
I don't think it is working.

https://github.com/openfun/marsha/pull/1744#discussion_r988290870